packages/types/src/sdk/datasources.ts

```diff
@@ -217,4 +217,21 @@ export interface DatasourcePlus extends IntegrationBase {
   ): Promise<Schema>
   getTableNames(): Promise<string[]>
   getViewNames?(): Promise<string[]>
+  getRelationships?(
+    tableNames?: string[]
+  ): Promise<DatasourceRelationshipConfig[]>
+}
+
+export enum DatasourceRelationshipType {
+  MANY_TO_ONE = "many-to-one",
```

Comment:
Why are you just defining this? Only `many-to-one` are supported with this feature?

Reply:
hey guys @adrinr  and @melohagan , i apologize, i answered this point via Visual Studio comments ( or so i thought )

the reasoning is that for every FK the PR creates both ends of the relationship but it skips self referencing ones

the implementation wont create many-to-many relationships, those must still be manually defined.

i used the typedef ( could be wrong ) to allow for future implementation of self referencing relationships.